src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/Books";
+    private static final String USER_NAME = "root";
+    private static final String USER_PASSWORD = "TelegramBOT";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", USER_NAME);
+        DB_PROPERTIES.put("password", USER_PASSWORD);
```

Comment:
I would suggest to make 'user' and 'password' literals constants rather than it`s values, because values could change dynamically, but words 'user' and 'password' will remain same